src/components/FocusButton.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useRef } from 'react';
+
+import { Tooltip, Button } from 'antd';
+import { EyeTwoTone, EyeOutlined } from '@ant-design/icons';
+import PropTypes from 'prop-types';
+import {
+  useSelector, useDispatch,
+} from 'react-redux';
+
+import { setCellInfoFocus } from '../redux/actions/cellInfo';
+
+const FocusButton = (props) => {
+  const { store, lookupKey, experimentId } = props;
+  const dispatch = useDispatch();
+
+  const focusData = useSelector((state) => state.cellInfo.focus);
```

Comment:
we will have different data for the FocusButton to display (gene expression when clicked from the gene table, cluster colouring when clicked on the cell set tool). Do we now put the cell set clustering info under `state.cellInfo.focus` when we want to focus on it?

Reply:
Correct. CellInfo now looks like this:
![image](https://user-images.githubusercontent.com/16019444/100613389-5c886980-330c-11eb-93e1-d2638f8d7ad0.png)

Reply:
cool, great!